packages/redux/MIGRATION.md

```diff
@@ -626,6 +624,13 @@ The following tables contain the exports that were removed and the alternatives
 
 The `useCheckout` hook was added to the `@farfetch/blackout-react` package which helps greatly when implementing a checkout feature in your apps. Prefer that instead of directly using the checkout actions/selectors.
 
+##### New helpers
+
+There are two new helpers to be imported from `@farfetch/blackout-redux`:
```

Comment:
I think this does not need to be added since these did not exist in v1. What do you think?

Reply:
Removed 👍